site/content/docs/include/common-svc-fields.en.md

```diff
@@ -240,13 +241,39 @@ Optional. The full config file path in your custom Fluent Bit image.
 
 <div class="separator"></div>
 
+<a id="publish" href="#publish" class="field">`publish`</a> <span class="type">Map</span>
+The `publish` section allows services to publish messages to one or more SNS topics. By default, no other services are allowed to subscribe to the created topics. Other services in the environment can be allowlisted using the `allowed_workers` field on each topic.
```

Comment:
Should we specify that it's only Worker Services that can subscribe to topics and not other type of services?
```suggestion
The `publish` section allows services to publish messages to one or more SNS topics. By default, no worker services are allowed to subscribe to the created topics. Worker Services in the environment can then be allowlisted using the `allowed_workers` field on each topic.
```